elyra/pipeline/component.py

```diff
@@ -255,7 +255,7 @@ def read_component_definition(self, component_id: str, location: str) -> str:
         component_location = os.path.join(os.path.dirname(__file__), location)
```

Comment:
(Sorry, GitHub wouldn't let me comment on the method definition line, so adding one here.)
Please update the return type hint to `-> Optional[str]:` since `None` can now be returned.